M2/Macaulay2/packages/ResLengthThree.m2

```diff
@@ -0,0 +1,752 @@
+newPackage ( "ResLengthThree",
```

Comment:
Could you add a comment at the top of the package about copyright and license? See the last bit of https://github.com/Macaulay2/M2/blob/master/M2/Macaulay2/packages/README